adapter/setEnvForAdapter.sh

```diff
@@ -0,0 +1,18 @@
+CONNECTION_RETRIES="30"
```

Comment:
Do we need this file? Better would be an env file...

Reply:
Oh I didn't see this file.
An env file is already added.
-> removed this script